lib/Elevate/Blockers.pm

```diff
@@ -146,6 +146,16 @@ sub is_check_mode ($) {
     return $_CHECK_MODE;
 }
 
+sub set_abort_on_first_blocker ( $self, $value ) {
```

Comment:
This probably just needs to be an accessor?

Reply:
use Simple::Accessor qw(set_abort_on_first_blocker).

Then just update the things that check `_abort_on_first_blocker`

Reply:
Unfortunately, we need the setter until we can do the refactor via RE-128.